tutorials/krusty/KRUSTY/openmc.i

```diff
@@ -0,0 +1,151 @@
+################################################################################
+## KRUSTY                                                                     ##
+## Heat Pipe Microreactor Steady State                                        ##
+##                                                                            ##
+## Mahmoud (3/25) (openmc.i/solid.i/model03.py/krusty_scale.e)                ##
+################################################################################
+
+power = 4000           # (W)
+
+[Mesh]
+  [fmg]
+    type = FileMeshGenerator
+    file = krusty_scale.e
+  []
+  # [scale]
```

Comment:
I'd remove anything which is commented out -- for part B, we can just add more input files. People often get confused by commented-out stuff